go/integration_test/interpreter/integration_test.go

```diff
@@ -1109,3 +1109,29 @@ func newMockStateDBForIntegrationTests(ctrl *gomock.Controller) *MockStateDB {
 
 	return mockStateDB
 }
+
+func TestEVM_CanSuccessfullyProcessPcBiggerThanCodeLength(t *testing.T) {
+	for _, variant := range Variants {
+		revision := Istanbul
+		t.Run(variant, func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			mockStateDB := newMockStateDBForIntegrationTests(ctrl)
+			// all implementations should be able to handle PC that goes beyond the code length.
+			// this is reproducible by not calling STOP, RETURN, REVERT or SELFDESTRUCT at the end of the code.
+			codeReturn := []byte{
```

Comment:
The name `codeReturn` is likely still from the code section this code was originally copied from. Please rename to something appropirate.